tutorials/plugins/editor/making_main_screen_plugins.rst

```diff
@@ -66,176 +47,92 @@ file. In our example, ``main_screen_plugin.gd``.
        pass
 
 
+    func handles(obj):
+       return false
+
+
     func get_plugin_name():
        return "Main Screen Plugin"
 
 
     func get_plugin_icon():
-       return get_editor_interface().get_base_control().get_icon("Node", "EditorIcons")
+       pass
```

Comment:
This function has a return type, so I think we should leave the example with the editor icon as is.